pyproject.toml

```diff
@@ -139,6 +139,7 @@ requires = [
   ignore = [
     "PLW2901", # Allow overwritten values on loops
     "PLR0915", # Allow condition check in list comprehension
+    "UP007", # Prefer Optional[], Union[] over | due to torch jit scripting
```

Comment:
is Jit supporting the others annotations from `__future__ import annotations`?

Reply:
Most of them yes, but not all of them. See: https://pytorch.org/docs/stable/jit_language_reference.html Though this page seems outdated, because I definitely had success with overloading functions in scripting.

Outside of `from __future__ import annotations`, the most notable thing missing is lack of support for `typing.Sequence`. Switching that to `Tuple`, `List` or some combination of both is an option, but that drops support for custom classes implementing sequence. Since, this is a possibly breaking change, and as mentioned in the issue, I'm not an expert in kornia ecosystem I decided to stop here.

Reply:
i see, I didn't expect jit to work with plain typing (`dict`, `list`, etc) since they stopped adding new typing options (or stopped to update the docs) -- i thought it was only working with typings `from typing import ...`